### DISCUSSION TITLE
Custom layout for error pages

### DISCUSSION THREAD

**Discussion by laborb-ch** [statamic/cms#5954] [Troubleshooting] [answered]
I'm currently creating custom error pages for a Statamic website.

I found out that for some reason the `404.antlers.html` template uses the default layout of the website, while the 503-template for maintenance mode uses no layout at all. Is it possible to define a specific layout for error templates or at least get a consistent behaviour?

Thanks in advance.
Cheers Chris

**Comment by duncanmcclean** [comment, marked as answer]
You can create a `resources/views/errors/layout.antlers.html` file which will be used instead of the main layout.

**Reply by laborb-ch** [reply]
This worked. Thanks for the quick response. :)
